Gemfile

```diff
@@ -48,3 +48,6 @@ end
 
 # Windows does not include zoneinfo files, so bundle the tzinfo-data gem
 gem 'tzinfo-data', platforms: [:mingw, :mswin, :x64_mingw, :jruby]
+
+# fix for the mimemagic no longer being supported
```

Comment:
Can you link the issue here and capitalize the word "fix"?

Reply:
link as in full url or just the issue number?

Reply:
link

Reply:
Updated